fastlane/fastlane
# [match] Fix skip_certificate_matching

### Checklist

### Motivation and Context


Using the `--skip_certificate_matching true` option with `fastlane match import` does not work as intended.
While it does not login in the match/import module, it still does in the storage implementations.

### Description


A parameter named `skip_spaceship_ensure` was added to applicable (all except git) storage implementations to prevent SpaceshipEnsure trying to log in.

### Testing Steps


Try running import with a team id to which you don't have access and use the `--skip_certificate_matching true` setting.
It should run without any login dialogs.
